doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -106,11 +106,10 @@ Enable partition mode in ACRN hypervisor
    .. code-block:: none
      :emphasize-lines: 1
 
-     $ dmesg | grep ttyS
-     [   10.552369] dw-apb-uart.8: ttyS4 at MMIO 0x91524000 (irq = 4, base_baud = 115200) is a 16550A
-     [   10.552568] serial serial0: tty port ttyS4 registered
-     [   10.572344] dw-apb-uart.9: ttyS5 at MMIO 0x91522000 (irq = 5, base_baud = 115200) is a 16550A
-     [   10.572545] serial serial1: tty port ttyS5 registered
+     $ sudo lspci | grep UART
+      00:18.0 . Series HSUART Controller #1 (rev 0b)
+      00:18.1 . Series HSUART Controller #2 (rev 0b)
+   The second with ``00:18.1`` is the one on the 40-pin expansion connector.
```

Comment:
You need an extra blank line and to start this sentence without any indentation.

Reply:
No.  What's written renders correctly.  This sentence is outside of the code block, but still within the numbered list body, so still needs to be indented (as it is).